language/stdlib/modules/Offer.move

```diff
@@ -35,6 +35,96 @@ module Offer {
   public fun address_of<Offered>(offer_address: address): address acquires T {
     borrow_global<T<Offered>>(offer_address).for
   }
+
+  /**
+   **************** SPECIFICATIONS ****************
+
+  This module defines a resource T that is used as a permissioned trading scheme between accounts.
```

Comment:
If this is general module doc and is not specific to specification, please move it before `module M {`.